### PR TITLE
Fixes IOS L3 intermittent zuul failure

### PR DESCRIPTION
**Pull request by justjais**
##### SUMMARY

Fixes IOS L3 intermittent zuul failure raised in bug #61305. Also, updated the tests IP to use documentation IPs instead of random.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

ios_l3_interfaces

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by justjais** [comment]
recheck

File: lib/ansible/module_utils/network/ios/config/l3_interfaces/l3_interfaces.py [hunk 1]
```diff
@@ -204,6 +204,34 @@ def _state_deleted(self, want, have):
 
         return commands
 
+    def check_diff_again(self, want, have):
+        """
+        Check the IPV4 difference again when secondary IP is configured,
```
Comment (danielmellado) [inline lib/ansible/module_utils/network/ios/config/l3_interfaces/l3_interfaces.py]: Could we rephrase this? I think it's a little bit confusing. IIUC when there's a difference in order there'd be a mismatch on the set? Too much difference on the comment ;)

**Review by danielmellado — APPROVED** [review]
Besides a minor comment on a test, it LGTM!

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
@justjais I've created an issue for the ios_file failure, so we can work on a fix:

https://github.com/ansible/ansible/issues/61694

**Review by pabelanger — CHANGES_REQUESTED** [review]
It looks like we still have an intermittent issue, you can see the last test failing:

  https://object-storage-ca-ymq-1.vexxhost.net/v1/a0b4156a37f9453eb4ec7db5422272df/ansible_82/61682/0b0550fc96f7fe814d6d39328993a79aa1cab1b0/third-party-check/ansible-test-network-integration-ios-python35/62fb746/controller/ara-report/

That happened because of:

  ios_l3_interfaces : Merge provided configuration with device configuration (IDEMPOTENT)

changing.

**Comment by justjais** [comment]
recheck

**Review by danielmellado — APPROVED** [review]
Looks like latest change got rid of the intermittent issue

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Review by danielmellado — APPROVED** [review]
Zuul seems happy after serveral rechecks, let's get this in!

**Pull request merged**